app/app/src/main/kotlin/com/hedvig/app/feature/marketing/MarketingActivity.kt

```diff
@@ -67,52 +72,43 @@ class MarketingActivity : AppCompatActivity() {
       }
       lifecycle.repeatOnLifecycle(Lifecycle.State.STARTED) {
         demoManager.isDemoMode().first { it == true }
-        activityNavigator.navigateToLoggedInScreen(this@MarketingActivity, false)
+        activityNavigator.navigateToLoggedInScreen(this@MarketingActivity, true)
```

Comment:
Not sure why it was not clearing the state before and what that actually meant practically. In any case however this doesn't matter here since I remove this completely anyway in a future PR